Upstream has a policy of only supporting one autoconf version with any
given PostgreSQL version; which is good for ensuring repeatable results
for PostgreSQL, but it's not very tenable in the Fedora/RHEL world.
Dike out the check.


diff -Naur postgresql-8.3.3.orig/configure.in postgresql-8.3.3/configure.in
--- postgresql-8.3.3.orig/configure.in	2008-06-08 20:38:40.000000000 -0400
+++ postgresql-8.3.3/configure.in	2008-07-28 21:45:08.000000000 -0400
@@ -19,10 +19,6 @@
 
 AC_INIT([PostgreSQL], [8.3.3], [pgsql-bugs@postgresql.org])
 
-m4_if(m4_defn([m4_PACKAGE_VERSION]), [2.59], [], [m4_fatal([Autoconf version 2.59 is required.
-Untested combinations of 'autoconf' and PostgreSQL versions are not
-recommended.  You can remove the check from 'configure.in' but it is then
-your responsibility whether the result works or not.])])
 AC_COPYRIGHT([Copyright (c) 1996-2008, PostgreSQL Global Development Group])
 AC_CONFIG_SRCDIR([src/backend/access/common/heaptuple.c])
 AC_CONFIG_AUX_DIR(config)
